src/components/course-header/styles.ts

```diff
@@ -0,0 +1,52 @@
+import { ThemeUICSSObject } from 'theme-ui';
+import theme from '@mdb/flora/theme';
+
+const CourseHeaderWrapper: ThemeUICSSObject = {
+  paddingTop: ['inc40', null, null, 'inc70'],
+  paddingBottom: ['inc60', 'inc50', null, 'inc70'],
+  paddingLeft: ['inc50', null, null, 'inc110'],
+  paddingRight: ['inc50', null, null, 'inc110'],
+  backgroundColor: theme.colors.blue80,
+};
+
+const CourseHeaderGrid: ThemeUICSSObject = {
+  gridGap: 0,
+  columnGap: [0, null, null, 'inc50'],
+};
+
+const CourseHeaderBreadcrumb: ThemeUICSSObject = {
+  marginBottom: 'inc40',
+  gridColumn: 'span 12',
+  'span.textlink-link-icon-class': {
+    lineHeight: ['inc20', 'inc30', null, null],
+  },
+};
+
+const CourseHeaderTitle: ThemeUICSSObject = {
+  gridColumn: ['span 12', null, null, 'span 7'],
+  marginBottom: ['inc40', null, null, 0],
+  maxWidth: '680px',
+};
+
+const CourseHeaderActions: ThemeUICSSObject = {
+  display: 'flex',
+  flexDirection: ['column', 'row', null, 'column'],
```

Comment:
So this array goes `mobileSmall, mobileLarge, tablet, desktopSmall, desktopLarge`. I think in order to have this go to `row` on tablet we should have `'column', null,' row', 'column'`.